### ISSUE TITLE
Add Millésime as a separate field or nutrient

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-server#1010]
Add Millésime as a separate field or nutrient for wines

**Comment by aleene** [comment]
Add to nutriments.txt?

**Comment by hangy** [comment]
What is a Millésime?

**Comment by CharlesNepote** [comment]
@hangy A millésime is the production year of a wine. https://de.wikipedia.org/wiki/Jahrgang_(Wein)

Millésime is also used by few specialists for other products, such as Sardines in boxes; see http://www.lefigaro.fr/lefigaromagazine/2008/03/29/01006-20080329ARTFIG00140-des-collectionneurs-fous-de-sardines-millesimees.php (french).

@aleene, @teolemon : why in nutriments.txt?!? why not use labels for that?

**Comment by aleene** [comment]
I agree, it has nothing to do with nutriments. Should be put in the date field.
